tests/test_range.py

```diff
@@ -724,10 +724,70 @@ def test_until(self):
         self.assert_range('until 20 hours from now', now, datetime(2017, 6, 17, 15))
         self.assert_range('until 45 minutes from now', now, datetime(2017, 6, 16, 20, 22))
         self.assert_range('until 45 seconds from now', now, datetime(2017, 6, 16, 19, 38, 7))
-        #
-        # # TODO "Until yesterday" etc: error or guess or infinity or unknown
-        # # TODO "Until the start of today" vs "Until the end of today"
 
+        # TODO "Until yesterday" etc: error or guess or infinity or unknown
+        # TODO "Until the start of today" vs "Until the end of today"
+
+    def test_week_start(self):
+        self.assert_range('this week',
+                          datetime(2017, 6, 11),
+                          datetime(2017, 6, 18),
+                          week_start=0)
+
+        self.assert_range('this week',
+                          datetime(2017, 6, 11),
+                          datetime(2017, 6, 18),
+                          week_start=7)
```

Comment:
Week start is 7 here, I don't know what that means but it seems to have no impact on the test case

Reply:
7 % 7 == 0 % 7 == 0